### ISSUE TITLE
Internal Media Player desperately needed

### ISSUE THREAD

**Issue by Jpersh27** [TeamAmaze/AmazeFileManager#2898]
Is your feature request related to a problem? 

• Yes

A clear and concise description of what the problem is.

• The file manager lacks an internal Media Player and at this point In development it desperately needs one. 

• It is frustrating and annoying to have to use an external application to play video files each and every time. It slows me down and requires me to use two applications at once instead of being able to view all my files inside one application.

**Describe the solution you'd like**

• A built-in Media player that recognizes and plays all current video file types and codecs that are widely used. 

**Describe alternatives you've considered**

• No alternative options to consider or have considered that would be similar.

**Additional context**

• If the concern is that you would like to implement an open source Media player to the application and have not found a suitable one may I suggest the following below as an option.

• https://github.com/google/ExoPlayer

• @VishalNehra @arpitkh96 @EmmanuelMess  and @TranceLove

**Comment by EmmanuelMess** [comment]
A media player is not something that should be in a File Manager, expect it to be an extension.

**Comment by VishalNehra** [comment]
Yes, it'll be a separate add-on. I can't give an ETA yet. Maybe few weeks.

**Comment by Jpersh27** [comment]
@VishalNehra it has been a few weeks and still no news or ETA on when this will be added as an external add-on???

**Comment by VishalNehra** [comment]
@Jpersh27 I've been caught up with stuff. Got absolutely no time. But I am bit free in upcoming 2 weeks. Let's hope I can finish in this time.

**Comment by Jpersh27** [comment]
@VishalNehra Thank you for the update I appreciate it and will be looking forward to hearing back from you soon.

@VishalNehra Can you also please consider reopening this case? https://github.com/TeamAmaze/AmazeFileManager/issues/2898 

• I feel as though that it was unnecessary for @EmmanuelMess to close it in the first place and feel like it would be a good way to track the progress of this issue/Add-on until it's released.
